app/notifications/validators.py

```diff
@@ -151,13 +150,9 @@ def time_until_end_of_day() -> timedelta:
     return datetime.combine(tomorrow, time.min) - dt
 
 
-def check_sms_limit_increment_redis_send_warnings_if_needed(
-    service: Service, requested_sms=0, key_type: ApiKeyType = KEY_TYPE_NORMAL
-) -> None:
+def check_sms_limit_increment_redis_send_warnings_if_needed(service: Service, requested_sms=0) -> None:
     if not current_app.config["FF_SPIKE_SMS_DAILY_LIMIT"]:
         return
-    if key_type == KEY_TYPE_TEST:
-        return
```

Comment:
We assume at this point that this check is made by the caller?

Reply:
Yes, I re-worked this logic so that this would not even be called in the case that a test key was used.